### PR TITLE
MODLOGSAML-131: Explain host name configuration

### PR DESCRIPTION
**Pull request by julianladisch**

### CODE REVIEW

**Review by adamdickmeiss — APPROVED** [review]
LGTM, but doesn't know much abt this. One could argue this should be part of mod-login-saml.. If not, I think there should be a link to this.

**Comment by julianladisch** [comment]
Yes, mod-login-saml should link to this.

File: content/en/docs/Getting started/Installation/hostnames.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+
+* https://folio-snapshot.dev.folio.org
+* https://folio-snapshot-okapi.dev.folio.org
+
+"Same site" has a special meaning, learn more at https://web.dev/samesite-cookies-explained/ and https://publicsuffix.org/ .
+
+Advantages:
+
+* Simple host name based configuration, requires no path based proxy configuration.
+* Continuously tested on FOLIOs reference environments.
```
Comment (steveellis) [inline content/en/docs/Getting started/Installation/hostnames.md:56]: Typo need apostrophe: "FOLIO's"

File: content/en/docs/Getting started/Installation/hostnames.md [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+FOLIO doesn't support running front-end and back-end on host names that belong to different sites.
+
+Fictional example:
+
+* https://beispiel.de
+* https://okapi.example.org
+
+This requires removing security configurations regarding cross-origin resource sharing (CORS) and cross-site request forgery (CSRF) and likely violates security rules required by law or policy.
+
+FOLIO prevents to configure it for the most security sensitive module, the single-sign-on (SSO) module mod-login-saml.
```
Comment (steveellis) [inline content/en/docs/Getting started/Installation/hostnames.md:74]: Suggested change: "FOLIO prevents this configuration for the most security sensitive module,..."

File: content/en/docs/Getting started/Installation/hostnames.md [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+
+Advantages:
+
+* Simple host name based configuration, requires no path based proxy configuration.
+* Continuously tested on FOLIOs reference environments.
+
+Disadvantages:
+
+* Slower because the browser needs to send extra "preflight" OPTIONS HTTP requests for cross-origin resource sharing (CORS) protection resulting in latency.
+* Less secure as it requires a configuration that lifts the security configurations regarding cross-origin resource sharing (CORS) and cross-site request forgery (CSRF).
```
[the remark this answers is not in the document]
Comment (julianladisch) [reply content/en/docs/Getting started/Installation/hostnames.md]: Better now?

**Review by steveellis — APPROVED** [review]
Looks good. Thanks for clarifying Julian!

File: content/en/docs/Getting started/Installation/hostnames.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (steveellis) [reply content/en/docs/Getting started/Installation/hostnames.md]: Yes, thanks!

**Pull request merged**